tests/st/libnetwork/test_assign_specific_ip.py

```diff
@@ -1,82 +0,0 @@
-# Copyright 2015 Tigera, Inc. All rights reserved.
```

Comment:
Whilst we do not have an explicit test case for this, a number of the calico-container Its do use specific IPs, so this is at least currently covered.